api/capi/include/tensor_filter_single.h

```diff
@@ -54,15 +55,7 @@ struct _GTensorFilterSingle
 {
   GObject element;     /**< This is the parent object */
 
-  void *privateData; /**< NNFW plugin's private data is stored here */
-  GstTensorFilterProperties prop; /**< NNFW plugin's properties */
-  const GstTensorFilterFramework *fw; /**< The implementation core of the NNFW. NULL if not configured */
-
-  /* internal properties for tensor_filter_single */
-  gboolean silent; /**< Verbose mode if FALSE. int instead of gboolean for non-glib custom plugins */
-  gboolean started; /**< filter has been started */
-  GstTensorsConfig in_config; /**< input tensor info */
-  GstTensorsConfig out_config; /**< output tensor info */
+  GstTensorFilterPrivate priv; /**< Internal properties for tensor-filter */
```

Comment:
Good!

Reply:
However, if the intention is to hide detailed from app writers, exposing tensor_filter_common.h is not good.